### PR TITLE
feat(github): github added to footer

### LINKED ISSUE #4028: Feature :Github link should be there in the footer with other social meadial links
### Is your feature request related to a problem? Please describe

there should be provided github link of CircuitVerse in the footer with other social media links google,facebook ,twitter so that it will be easy for user to find github repo 
Screenshot (269)
.
.
.
like this 
Screenshot (263)

### Describe the solution you'd like

for this we should also provide git hub repo link in footer like social media links 

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by harshsharma-11**
Fixes #4028 

#### Describe the changes you have made in this PR -
added a github repository link in footer 

### Screenshots of the changes (If any) -

Screenshot (277)

updated ;;;;;;;;;;;;;;;;;;;;;;;;;;;
.
.
.
.

Screenshot (287)
Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -16,6 +16,7 @@
           <a class="bounce-out-on-hover" href="/facebook" target="_blank" data-tooltip="facebook"><%= image_tag("logos/facebook-logo.png", class: "footer-social-icon pt-2", alt: "Facebook Logo") %></a>
           <a class="bounce-out-on-hover" href="/twitter" target="_blank" data-tooltip="twitter"><%= image_tag("logos/twitter-logo.png", class: "footer-social-icon pt-2", alt: "Twitter Logo") %></a>
           <a class="bounce-out-on-hover" href="/linkedin" target="_blank" data-tooltip="linkedin"><%= image_tag("logos/linkedin-logo.png", class: "footer-social-icon pt-2", alt: "LinkedIn Logo") %></a>
+          <a class="bounce-out-on-hover" href="https://github.com/CircuitVerse" target="_blank" data-tooltip="github"><%= image_tag("logos/github-logo-square.jpg", class: "footer-social-icon pt-2", alt: "GitHub Logo") %></a>
```
Comment (Arnabdaz) [inline app/views/layouts/_footer.html.erb:19]:
```suggestion
          <a class="bounce-out-on-hover" href="/github" target="_blank" data-tooltip="github"><%= image_tag("logos/github-logo-square.jpg", class: "footer-social-icon pt-2", alt: "GitHub Logo") %></a>
```

**Review by Arnabdaz — CHANGES_REQUESTED** [review]
1. use the `/github` route instead of the actual url 
image

2. Use a icon with transparent background.. ie, it should not have the white square edges and should be round edged

**Comment by tanmoysrt** [comment]
@harshsharma-11 remove the background of github logo, as that will not look good in dark background

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (harshsharma-11) [reply app/views/layouts/_footer.html.erb:19]: okk let me make these changes

Comment (harshsharma-11) [reply app/views/layouts/_footer.html.erb]: @Arnabdaz done please check

**Review by Arnabdaz — APPROVED** [review]
LGTM 👍 

cc @tachyons

**Review by Arnabdaz — APPROVED** [review]

**Comment by harshsharma-11** [comment]
@tachyons please kindly review the pr as discuss in #4028

**Review by tachyons — CHANGES_REQUESTED** [review]
Please add logo with transparent background for consistency

**Comment by harshsharma-11** [comment]
cc@tachyons 
Screenshot (287)

**Comment by tanmoysrt** [comment]
> cc @tachyons Screenshot (287)

update this image in PR description as well

**Review by Arnabdaz — CHANGES_REQUESTED** [review]
@harshsharma-11 use logo from https://github.com/logos as current logo does not have a transparent background. else in future logo would have to be updated in case of change in footer color scheme

**Comment by harshsharma-11** [comment]
@tachyons @Arnabdaz please check i made bg transparent
Screenshot (288)

**Pull request merged**